packages/paste-website/src/pages/introduction/for-designers/design-guidelines/index.mdx

```diff
@@ -60,53 +56,136 @@ account through [ServiceNow](https://twilio.service-now.com/sp/). When you have
+      File and Cover Sheet Template
+    </Anchor>
+  </ListItem>
+</UnorderedList>
+
+<Callout variant="neutral">
+  <CalloutHeading as="h5">👉 Our recommendations for using the new Figma libraries</CalloutHeading>
+  <CalloutText>
+    Use the new libraries for net-new designs, rather than trying to swap older design files to the new libraries, as
+    things could be detached and our new components library has some updated structuring.
+  </CalloutText>
+  <CalloutText>
+    We do not recommend files that use the [SUNSET] Paste components to swap to the new Twilio or dark themes. It would
+    be better to first try to swap your file that uses [SUNSET] Paste Components to Twilio Components. We’ve updated
+    some of our components structuring and tokens that wouldn’t get applied cleanly or accurately.
+  </CalloutText>
+</Callout>
+
+<Box paddingBottom="space70" />
+
+#### Product-specific libraries
+
+- [Console Library](https://www.figma.com/file/vvdKgsO8OLcn7Q6FmuhWtl/Console-Library) for teams with Console projects (managed by the Console team)
+- [Evergreen Paste Components](https://www.figma.com/file/zq39AEOAd7EvvB1gvL8wjE/%5BNEW%5D-Evergreen-Paste-Components?t=LqMbG6udvN37EeWy-6), [Paste Evergreen Theme](https://www.figma.com/file/nqTRnOvgtvJyGw5loSAZWr/%5BWIP%5D-Paste-Evergreen-Theme?t=yTgj99DMtixcB9l5-6), [Evergreen Paste Patterns](https://www.figma.com/file/kfIKmexsAIaU3gjdnBVY17/Evergreen-Paste-Patterns?t=Cw7YRtCJ0ILxlKD4-6) includes components, tokens, and patterns that look like Evergreen. You can use this library until your product migrates to the Paste Components library.
 
 ---
 
 ## Using themes
 
-Paste currently has [Default Theme](https://www.figma.com/file/OZKRQFOkOAl413m5JVIYE6/Paste-Default-Theme) and
-[Dark Theme](https://www.figma.com/file/TZMBaCxXkX8u5aVRVYboid/Paste-Dark-Theme?node-id=0%3A1) libraries.
+Paste uses 2 theme libraries.
+
+- [Paste Twilio Theme](https://www.figma.com/file/U0oTLE09SGK7svnr818HYT/%5BWIP%5D-Paste-Twilio-Theme?node-id=3558%3A32533&t=PYbe5AovStxUbEjF-1)
+- [Paste Twilio Dark Theme](https://www.figma.com/file/l2qtpQVMmXzQPVB3rEAA8i/%5BWIP%5D-Paste-Components?node-id=3015%3A46646&t=2Fe524au9O9rlWZU-1)
 
 When you have a theme library linked, you'll have access to:
 
 - Text styles (font families, font weights, and font sizes)
-- Colour styles (fill and stroke for background, border, and text colors)
-- Effect styles (shadows, etc.)
+- Color styles (fill and stroke for background, border, and text colors)
+- Effect styles (drop shadows, etc.)
 
 ![Menu in Figma when you can navigate through tokens for the Default theme](../../../../assets/images/design-guidelines/token-styles.png)
 
-You should be using components from Paste to compose your UIs whenever possible, but the text and layer styles will be helpful when you need to compose something more custom.
-
-This can be anything from setting a piece of text next to an icon to creating a custom stepper component.
-
 ---
 
 ## Using components
```

Comment:
todo: Remove the lines "Any layer with a ⚙️ emoji indicates that there are nested variants available." and "Any layer with a ✏️ emoji indicates an editable layer."